Index: src/__init__.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from .models import Graph\r\nfrom .models import Node\r\nfrom .models import PaperNode\r\nfrom .models import GreedyCohesiveClustering\r\nfrom .data_processing import process_pdfs\r\nfrom .data_processing import Encoder\r\n# to avoid runtime error of CLI call for double imports\r\n# from .data_processing import generate_embeddings\r\n# from .data_processing import csv_to_paper_node\r\nfrom .config import PROJECT_ROOT, DATA_DIR\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/__init__.py b/src/__init__.py
--- a/src/__init__.py	(revision 8ed3e4d34e627d97a3b7bf573bcde02acf193721)
+++ b/src/__init__.py	(date 1747331747003)
@@ -5,7 +5,7 @@
 from .data_processing import process_pdfs
 from .data_processing import Encoder
 # to avoid runtime error of CLI call for double imports
-# from .data_processing import generate_embeddings
-# from .data_processing import csv_to_paper_node
+from .data_processing import generate_embeddings
+from .data_processing import csv_to_paper_node
 from .config import PROJECT_ROOT, DATA_DIR
 
Index: src/data_processing/__init__.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from .pdfs_to_text import process_pdfs\r\nfrom .encoder import Encoder\r\n# to avoid runtime error of CLI call for double imports\r\n# from .csv_to_paper_node import main\r\n# from .generate_embeddings import main
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/data_processing/__init__.py b/src/data_processing/__init__.py
--- a/src/data_processing/__init__.py	(revision 8ed3e4d34e627d97a3b7bf573bcde02acf193721)
+++ b/src/data_processing/__init__.py	(date 1747331746987)
@@ -1,5 +1,5 @@
 from .pdfs_to_text import process_pdfs
 from .encoder import Encoder
 # to avoid runtime error of CLI call for double imports
-# from .csv_to_paper_node import main
-# from .generate_embeddings import main
\ No newline at end of file
+from .csv_to_paper_node import main
+from .generate_embeddings import main
\ No newline at end of file
